### DISCUSSION TITLE
best config postal server

### DISCUSSION THREAD

**Discussion by mredodos** [postalserver/postal#2665] [Help with using Postal]
in your repository i have see this example file  https://github.com/postalserver/postal/blob/main/config/postal.defaults.yml

Which of that settings is important to add to our config file?

**Comment by willpower232** [comment]
The absolute minimum is part of the install repo here https://github.com/postalserver/install/blob/main/examples/postal.yml

It gets combined with the file you linked to so that one acts like setting all the other defaults and is a good reference if you need to add more things to your config file because of your environment or use case.

**Reply by mredodos** [reply]
ok thx, but i think is better if you add more documentation about that because a lot of this stuff is not clear and is not simple with out doc
